src/Shared/TracingHelper.cs

```diff
@@ -32,7 +32,7 @@ internal static class TracingHelper
 
     public static Activity? StartConsumerActivity(string activityName, ActivityContext linkedActivityContext)
     {
-        var activityLinks = new[] { new ActivityLink(linkedActivityContext) };
+        var activityLinks = linkedActivityContext == default ? Enumerable.Empty<ActivityLink>() : [new ActivityLink(linkedActivityContext)];
```

Comment:
todo: validate behavior of `GivenActivityListener_WhenHandleEventGridEvent_ThenHandleWithTracing` test; whether the links is valid or not. There is a chance that the links is empty.

Reply:
resolved by fixing the test